ci-operator/step-registry/hypershift/openstack/create/hostedcluster/hypershift-openstack-create-hostedcluster-commands.sh

```diff
@@ -30,14 +33,34 @@ COMMAND=(
   --openstack-external-network-id "${OPENSTACK_EXTERNAL_NETWORK_ID}"
   --openstack-node-flavor "${OPENSTACK_COMPUTE_FLAVOR}"
   --openstack-node-image-name "${RHCOS_IMAGE_NAME}"
+  --node-upgrade-type InPlace
```

Comment:
I wonder could we get a comment above the command explaining this? The rest of these options are pretty self-explanatory IMO.

Reply:
good point